composer.json

```diff
@@ -3,27 +3,29 @@
     "description": "MyComponent description",
     "license": "MIT",
     "require": {
-        "php": "^8.1",
-        "keboola/php-component": "^9.0"
+        "php": "^8.2",
+        "keboola/csv": "^1.5",
+        "keboola/php-component": "^9.0",
+        "keboola/storage-api-client": "^14.9"
     },
     "require-dev": {
-        "php-parallel-lint/php-parallel-lint": "^1.3",
-        "keboola/coding-standard": ">=7.0.2",
-        "keboola/datadir-tests": "^5.3",
+        "keboola/coding-standard": "^14.0",
+        "keboola/datadir-tests": "^5.5",
         "keboola/php-temp": "^2.0",
-        "phpstan/phpstan": "^1.4",
-        "phpunit/phpunit": "^9.5",
-        "symfony/process": "^5.0"
+        "php-parallel-lint/php-parallel-lint": "^1.3",
+        "phpstan/phpstan": "^1.10",
+        "phpunit/phpunit": "^9.6",
+        "symfony/process": "^6.3"
     },
     "autoload": {
         "psr-4": {
-            "MyComponent\\": "src/"
+            "Keboola\\MergeBrancheStorage\\": "src/"
```

Comment:
Tady bych taky v těch namespacech fixnul to typo (i u testů)

Reply:
jj... je to fixnutý